JS-Closures.md

```diff
@@ -1,6 +1,6 @@
 # JavaScript Closures
 
-Closures are functions that refer to independent (free) variables. In other words, the function defined in the closure 'remembers' the environment in which it was created.
+Closures are functions that refer to independent (free) variables. Free variables are variables that are used locally, but defined in an enclosing scope. In other words, the function defined in the closure 'remembers' the environment in which it was created.
```

Comment:
> Closures are functions that refer to independent (free) variables.

```javascript
var name = "Apple";
function displayName() {
     alert(name);
}
```
This is a function that refers to a free variable, but it is not a closure because it is not returned by an enclosing function. Therefore, I think the first sentence should read:

> Closures are functions, that refer to independent (free) variables, that have been returned by an enclosing function.